### PR TITLE
[REF] odoo-shippable: Add standard packages

### PR DESCRIPTION
**Pull request by moylop260**

### CODE REVIEW

**Comment by ruiztulio** [comment]
@moylop260 Better install the github version, the one from oficial repo is a little outdated: https://github.com/dalibo/pgbadger

**Comment by moylop260** [comment]
@ruiztulio 
Ok I got it
Thank you

@hugho-ad 
Could you change this PR to add pgbadger installation and basic dummy (disabled) configuration?

File: odoo-shippable/Dockerfile [hunk 1]
```diff
@@ -106,8 +106,9 @@ RUN pip install pylint_odoo
 #  Setting global env for next shippable build
 ENV WITHOUT_DEPENDENCIES 1
 
-# Install curl standard package to download files
-RUN apt-get install curl
+# Install standard packages
+RUN apt-get install -y curl htop p7zip-full pgbadger \
```
Comment (ruiztulio) [inline odoo-shippable/Dockerfile]: @moylop260 Curl is already installed in the base ubuntu image: https://github.com/Vauxoo/docker-ubuntu-base/blob/master/Dockerfile#L23

(sorry for my late reply)

Comment (moylop260) [inline odoo-shippable/Dockerfile]: I got it
Thank you!
curl and htop removed from odoo-shippable

**Comment by moylop260** [comment]
- [x] Removed packages installed from base image.
- [x] Remove pgbadger @hugho-ad will work it in other pr

**Pull request merged**